tasks/main.yml

```diff
@@ -5,6 +5,22 @@
   template: src=timesyncd.conf dest=/etc/systemd/timesyncd.conf
   notify: Enable ntp
 
+- name: create systemd drop-in directory
+  file: path={{docker_dropin_dir}} state=directory mode=0755
+  sudo: yes
+  when: setup_logging
+
+- name: install logging env
+  sudo: yes
+  template: src=docker-logging.env dest=/run/docker-logging.env
```

Comment:
`/run` is not a proper location, as it's going to be cleaned up next time we reboot (this is a `tmpfs` mountpoint)